employees/common/exports.py

```diff
@@ -0,0 +1,140 @@
+from openpyxl import Workbook
+from openpyxl.styles import Alignment
+from openpyxl.styles import Font
+from openpyxl.utils import get_column_letter
+
+from employees.common.constants import ExcelGeneratorSettingsConstants
+from managers.models import Project
+from users.models import CustomUser
+
+
+def set_format_styles_for_main_cells(cell):
```

Comment:
You are also missing typings in this pull request.

Reply:
I think done but I am not sure about typings to `worksheet`, `workbook` `cell`.

Reply:
Check if mypy check passes without warnings :)

Reply:
There are no warning but I gave them object type:P

Reply:
Nice try :) The correct types should be in `openpyxl` library which you have installed in venv (btw. you can set venv for project in Pycharm, ten ctrl+click on something will take you to the source code), so you can check them there.

Reply:
Done:)